packages/expo-web-browser/ios/WebAuthSession.swift

```diff
@@ -31,6 +31,9 @@ final internal class WebAuthSession {
         ])
       }
     )
+    if #available(iOS 13.0, *) {
+      self.authSession?.prefersEphemeralWebBrowserSession = options.preferEphemeralSession
+    }
```

Comment:
Maybe I should move this to the `open()` func right below 🤔

Reply:
up to you 🤷‍♂️